scripts/partition_manager.py

```diff
@@ -372,10 +372,26 @@ def _set_addresses_and_align(reqs, sub_partitions, solution, size, start, dynami
         if align_if_required(i, dynamic_partitions, False, reqs, solution):
             _set_addresses_and_align(reqs, sub_partitions, solution, size, start, dynamic_partitions)
 
+def align_is_valid(align):
```

Comment:
IMO the name sounds like it should return a bool, not raise an error. `verify_align_is_valid`

Reply:
I was actually thinking the same, but decided to not comment.
So I fully agree with this comment.